Sources/XCTest/Public/XCTestCase.swift

```diff
@@ -16,7 +16,9 @@
 /// This type is intended to be produced by the `testCase` helper function.
 /// - seealso: `testCase`
 /// - seealso: `XCTMain`
-public typealias XCTestCaseEntry = (testCaseClass: XCTestCase.Type, allTests: [(String, (XCTestCase) throws -> Void)])
+public typealias XCTestCaseEntry = (testCaseClass: XCTestCase.Type, allTests: [(String, XCTestCaseClosure)])
+
+public typealias XCTestCaseClosure = (XCTestCase) throws -> Void
```

Comment:
Sorry, a bit of a nit-pick: how about moving this `typealias` above the first, so that readers don't encounter `XCTestCaseClosure` before it's defined?

Also, this typealias could use some documentation.

Reply:
I totally agree!

Let me know if the documentation is helpful or it needs improvements.